website/src/logged_in/logs/Logs.js

```diff
@@ -76,6 +76,147 @@ const columns = [
             }
         }
     },
+    {
+        field: 'minF&B',
+        headerName: 'Min F&B Per Contract',
+        description: 'F&B Minimum Per Contract',
+        sortable: true,
+        width: 200,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['F&B Minimum Per Contract']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'area',
+        headerName: 'SQFT per Event',
+        description: 'SQFT per Event',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['SQFT per Event']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'type',
+        headerName: 'Type',
+        description: 'Event Type',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['event_type']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'type-account',
+        headerName: 'Type-account',
+        description: 'Event Type account',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['type_account']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'totalDays',
+        headerName: 'Total Days (In-Out)',
+        description: 'Total Days (In-Out)',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['total_days']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'isTelecom',
+        headerName: 'isTelecom',
+        description: 'isTelecom',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['is_telecom']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'isAudio',
+        headerName: 'isAudio',
+        description: 'isAudio',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['is_audio']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'eventDaynumber',
```

Comment:
What data does this column provide? Are we not already capturing this elsewhere? Is this for total event duration in days?

Reply:
If so, then the text needs to be rewritten and cleaned up.

Reply:
It is not. It actually represents the date during a year. For instance, if the event happens on January 7th, then the eventDaynumber is 7. If the event happens on February 1st, then the event Daynumber is 32. I try to find a clearer name for it but can't think of any.

Reply:
I will delete it. It is redundant.